### DISCUSSION TITLE
Can we merge now?

### DISCUSSION THREAD

**Discussion by YanLi2024** [BenjaminKey/Tafe24S1T105_Ext#5] [General]
Hi Ben
I have pushed to remote repository. If there is no change from your side, can we start to merge now?

**Comment by BenjaminKey** [comment]
Hi Yan
I am ready to merge.
If you want to merge first, I will handle any conflicts.
Happy to do it the other way round also.

we can have a look at the removal of buttons when merged back into the dev branch

**Reply by YanLi2024** [reply]
I did merge. No conflicts happened.

**Reply by BenjaminKey** [reply]
Hi Yan
Did you merge your Currency converter or my Mortgage calculator.
The commit comment says mortgage calc. 

I cant seem to access the visual studio project in my local repo on the development branch now.
not sure how to fix. what is your thoughts?

**Reply by YanLi2024** [reply]
Hi Ben
I merged both features since there is no conflicts to work on. :)
I just cloned, opened it and checkout development branch. It works on my computer.

**Comment by BenjaminKey** [comment]
Hi Yan

I have the local repo working now - visual studio required an update.
The button changes look good!

I have moved on to section D and have completed the merge to the release branch.
Feel free to continue on with section D.
![image](https://github.com/BenjaminKey/Tafe24S1T105_Ext/assets/160720574/6efffbce-e184-4d2e-ab03-65952fc6fc50)

**Comment by YanLi2024** [comment]
Hi Ben
I finished section D, a bit different from your revision graph.
You have a master branch at the bottom, wondering if I missed any step? 

![image](https://github.com/BenjaminKey/Tafe24S1T105_Ext/assets/160372202/ee4933df-1876-468b-bc68-888604b71c4f)

According to the requirements in Section D, we should design the main menu page now but we already completed before merge. So we could just ignore this step? 

Can KT read all our discussions? didn't see his response to your calling for help.

**Reply by BenjaminKey** [reply]
Hi Yan

I definitely had a misstep in mine, but I couldn't change it.
Not sure if KT moniters the discussion, so I just kept on going.
I have now got the release branch on the github repo(sorry, thought I had done that), that is the one that needs to be merged back into the main branch. Release branch is 3 commits ahead, see below.
This merge will complete Section D, did you want to do the final merge?

![image](https://github.com/BenjaminKey/Tafe24S1T105_Ext/assets/160720574/e97614d8-3c35-4429-8c94-ffae86858d19)

**Comment by YanLi2024** [comment]
Hi Ben
I think it's fine, you can do the final merge if you want. I clone the files each time I continue this assignment so that I know if you have made any changes.

Do we need to rename the local base branch name as master at this stage and leave the remote one as main?

o	**Push the master branch to the remote main _branch**

I am not sure about this statement. How do you think?
![image](https://github.com/BenjaminKey/Tafe24S1T105_Ext/assets/160372202/6d1c44d1-199d-40d2-8caa-590e0a53726a)
